Flow.Launcher/MainWindow.xaml

```diff
@@ -46,6 +46,7 @@
         <KeyBinding Key="Home" Modifiers="Alt" Command="{Binding SelectFirstResultCommand}"></KeyBinding>
         <KeyBinding Key="O" Modifiers="Ctrl" Command="{Binding LoadContextMenuCommand}"></KeyBinding>
         <KeyBinding Key="H" Modifiers="Ctrl" Command="{Binding LoadHistoryCommand}"></KeyBinding>
+        <KeyBinding Key="Enter" Modifiers="Ctrl+Shift" Command="{Binding OpenResultCommand}"></KeyBinding>
```

Comment:
Is this needed?

Reply:
yes since CTRL+Enter and Shift+Enter are already bound each on its own, if you press both CTRL and Shift the command will not be triggered unless we add an explicit binding for both being pressed at the same time.